app/assets/sass/modules/_breadcrumbs.scss

```diff
@@ -0,0 +1,31 @@
+.breadcrumbs {
```

Comment:
Actually, i'll just leave it since we moving it out soon on this ticket https://trello.com/c/MBBjKGnd/43-spike-how-we-could-copy-the-styles-over-from-the-live-site-s-or-link-directly-to-the-production-css-files